concent-builder/configure.yml

```diff
@@ -73,6 +73,8 @@
             - libpq-dev
             - gcc
             - python3-yaml
+            - postgresql-9.6
+            - python-psycopg2
```

Comment:
I think you should read the PostgreSQL version from `versions.yml`.

Reply:
Done.